src/test/rustdoc/auxiliary/macro-2-reexport.rs

```diff
@@ -0,0 +1,6 @@
+#![crate_name = "macro_2_reexport"]
```

Comment:
nit: Isn't this `crate_name` redundant? Is it just so that renaming the test doesn't cause it to fail?

Reply:
It's completely redundant but I find it easier this way to have the crate name written even if useless.